### DISCUSSION TITLE
peer selector for data attribute

### DISCUSSION THREAD

**Discussion by ahmadghoniem** [tailwindlabs/tailwindcss#11903] [Help] [answered]
i'm trying to do something like this 
here's a playground link: https://play.tailwindcss.com/60GfIZXRZK
![image](https://github.com/tailwindlabs/tailwindcss/assets/60299751/a80043fb-1743-4e4d-92da-56c95b37a7d2)
it's working with arbitrary values like this 
![image](https://github.com/tailwindlabs/tailwindcss/assets/60299751/a1eaf901-0930-4488-85d7-40fd28826032)
but not working with what i'm trying to do

**Comment by wongjn** [comment, marked as answer]
You'd use a [`peer-data-*` arbitrary variant](https://tailwindcss.com/docs/hover-focus-and-other-states#data-attributes) like `peer-data-[state=on]:` (where the string in brackets is of the form `[attribute-name-without-data-prefix=value]`) or an [arbitrary `peer-*` variant](https://tailwindcss.com/docs/hover-focus-and-other-states#arbitrary-peers) like `peer-[[data-state=on]]:` (where the string in brackets is a CSS selector).

[Tailwind Play](https://play.tailwindcss.com/rsLz0vScxD)

**Reply by ahmadghoniem** [reply]
Great thank you!

**Comment by PickleNik** [comment]
Should `peer-data-[...]` syntax be documented[ here?](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state)

Or in [quick reference](https://tailwindcss.com/docs/hover-focus-and-other-states#quick-reference) since it lists `peer-[...]` and `data-[...]` but it is unclear if and how you can nest them. I was trying to nest them through `:` until I found this post.
